lib/final-api/v1/http/ddtf_build.rb

```diff
@@ -124,14 +124,13 @@ def atom_response
         end
 
         def execution_logs
-          execution_logs_sorted = build.execution_logs.order(:position)
-
-          messages = execution_logs_sorted.map do |message|
-            dateformat = message[:timestamp] ?  message[:timestamp].strftime("%d.%m.%Y %H:%M:%S") : 'unknown date'
-            "#{dateformat}: #{message[:message]}"
+          build.execution_logs.each_with_object([]) do |execution_log, result|
```

Comment:
Maybe we could add unit test to verify that array of hashes with these 3 fields is returned?